src/editor/actions/autosave-actions.js

```diff
@@ -0,0 +1,64 @@
+import debounceAction from "debounce-action";
```

Comment:
can you annotate this file to indicate which parts are just things moved over from actions.js and what is new?